repose-aggregator/core/valve/src/test/scala/org/openrepose/valve/ReposeJettyServerTest.scala

```diff
@@ -199,7 +199,5 @@ class ReposeJettyServerTest extends FunSpec with Matchers with BeforeAndAfterAll
     expectedProperties.foreach { case (k, v) =>
       server.appContext.getEnvironment.getProperty(k) shouldBe v
     }
-
   }
-
```

Comment:
Might as well revert this file to master. With the route we're going, this change is unrelated.